### PR TITLE
Functions added to obtain comments via review_id

### PR DESCRIPTION
**Pull request by JLaw96**
Functions have been added to obtain the comments via the review_id. The app.test file has also been updated accordingly and errors have been taken into account.

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -175,4 +176,56 @@ describe("app", () => {
...
+        .get("/api/reviews/3/comments")
+        .expect(200)
+        .then(({ body }) => {
+          const { comments } = body;
+          console.log(comments, "I am comments in the test file");
+          expect(comments).toHaveLength(3);
+          expect(comments).toBeInstanceOf(Array);
+          expect(comments).toBeSorted({ descending: true });
+        });
+    });
```
Comment (Keb84) [inline __tests__/app.test.js:209]: you could add a test for 200 : valid ID but no comments and so responds with an empty array

**Review by Keb84 — APPROVED** [review]
There's one more test to add here - (see comments) then feel free to merge 👍
